### ISSUE TITLE
traverse display null elements in edges but browse class shows them

### ISSUE THREAD

**Issue by gg4u** [orientechnologies/orientdb#4816]
Hi,
following step by step tutorial:
https://orientdb.com/docs/last/Tutorial-Using-schema-with-graphs.html

Following traversal display null results for me, while I can see edges are not null in 'browse class'. 
Could you help in understanding why?

``` sql
SELECT NAME FROM ( SELECT EXPAND( OUT( 'Owns' ) ) FROM Person WHERE name = 'Luca' )

 ----+-------+----------------
  #  | @RID  | name
 ----+-------+----------------
  0  | #-2:1 | Ferrari Modena
 ----+-------+----------------
```

<img width="568" alt="screen shot 2015-08-18 at 12 34 34 pm" src="https://cloud.githubusercontent.com/assets/284214/9327999/1fcec2b4-45a5-11e5-90a7-ef3dd281081c.png">

**Comment by lvca** [comment]
What OrientDB version are you using?

**Comment by gg4u** [comment]
2.1.0

**Comment by santo-it** [comment]
Hi,

Thanks for opening this issue. Verified with 2.2.18.

The issue is caused by the fact that name is uppercase in the select but it was created in lowercase (with the syntax CREATE VERTEX Car SET name = 'Ferrari Modena').

I have fixed the select in the manual (version 2.2. and the 3.0-snapshot).

We are working to improve, in new major versions, intuitiveness of case sensitivity with OrientDB SQL.
We are also working to have test cases for the documentation examples.

Thanks,
